Sources/Plasma/PubUtilLib/plAudio/plWin32StaticSound.cpp

```diff
@@ -171,6 +182,19 @@ bool plWin32StaticSound::LoadSound( bool is3D )
             return false;
         }
 
+        // check if subtitles are enabled and if srcFilename is a localized audio file (e.g., ending in _eng, _fre, etc.)
+        // TODO: surely there is already a function somewhere to do this localization filename check?
+        if (plgAudioSys::IsEnabledSubtitles() && std::regex_match(srcFilename.StripFileExt().AsString().c_str(), std::regex("^.*_(eng|fre|ger|spa|ita|jpn)$", std::regex_constants::icase))) {
```

Comment:
Couple of things:
- If I'm reading this correctly, the first leg of this if statement will prevent changing the subtitle on/off state from working until relinking. IMO, the subtitles should always be loaded and we should just check the on/off state before dispatching the subtitle message.
- As your comment notes, there is a better way to do the second leg. Just check to see if the sound buffer filename is localized using `plLocalization::GetLocalized()`.

Reply:
It would no work to turn on/off subtitles in the middle of an already playing audio, but I'm not sure why you're saying it would prevent changing the enabled state via console or options menu until the player links to a different Age? I might be misunderstanding. I guess it would be nicer for it to check at dispatch-time. I was just trying to prevent incurring extra overhead (albeit relatively small overhead, since the SRT files are fairly small) for players who play with the "default" no-subtitle setting.

Reply:
`LoadSound()` bails if the sound is already loaded, so you would need to link out and back in to force `LoadSound()` to proceed. I would favor better UX over optimizing for the handful of kilobytes memory saved by not parsing the subtitle file.